tfjs-backend-webgpu/src/webgpu_program.ts

```diff
@@ -65,6 +65,27 @@ export const compileProgram =
         layout: 'auto'
       });
 
+      let printShaderString = env().get('WEBGPU_PRINT_SHADER') as string;
+      if (printShaderString === '') {
+        return pipeline;
+      }
+      // If set WEBGPU_PRINT_SHADER to boolean or number, throw error.
+      try {
+        printShaderString = printShaderString.toLowerCase();
+      } catch {
+        throw new Error(
+            `WEBGPU_PRINT_SHADER doesn't support value ` +
+            `'${printShaderString}'. ` +
+            `It should be 'all' or program name`);
```

Comment:
How can the user know the supported `program name`?

Reply:
I changed this to shader key.